allennlp/tests/modules/token_embedders/bert_embedder_test.py

```diff
@@ -0,0 +1,174 @@
+        batch.index_instances(vocab)
+
+        padding_lengths = batch.get_padding_lengths()
+        tensor_dict = batch.as_tensor_dict(padding_lengths)
+        tokens = tensor_dict["tokens"]
+
+        assert tokens["bert"].tolist() == [
+                [2, 3, 4, 3, 5, 6, 8, 9, 2, 14, 12, 0],
+                [2, 3, 5, 6, 8, 9, 2, 15, 10, 11, 14, 1]
+        ]
+
+        assert tokens["bert-offsets"].tolist() == [
+                [0, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                [0, 1, 2, 3, 4, 5, 6, 9, 10, 11]
+        ]
+
+        # No offsets, should get 12 vectors back.
+        bert_vectors = self.token_embedder(tokens["bert"])
+        assert list(bert_vectors.shape) == [2, 12, 12]
+
+        # Offsets, should get 10 vectors back.
+        bert_vectors = self.token_embedder(tokens["bert"], offsets=tokens["bert-offsets"])
+        assert list(bert_vectors.shape) == [2, 10, 12]
+
+    def test_padding_for_equal_length_indices(self):
+        tokenizer = WordTokenizer(word_splitter=BertBasicWordSplitter())
+
+        #            2   3     5     6   8      9    2   14   12
+        sentence = "the quick brown fox jumped over the lazy dog"
+        tokens = tokenizer.tokenize(sentence)
+
+        vocab = Vocabulary()
+
+        instance = Instance({"tokens": TextField(tokens, {"bert": self.token_indexer})})
+
+        batch = Batch([instance])
+        batch.index_instances(vocab)
+
+        padding_lengths = batch.get_padding_lengths()
+        tensor_dict = batch.as_tensor_dict(padding_lengths)
+        tokens = tensor_dict["tokens"]
+
+        assert tokens["bert"].tolist() == [
+                [2, 3, 5, 6, 8, 9, 2, 14, 12]
+        ]
+
+        assert tokens["bert-offsets"].tolist() == [
+                [0, 1, 2, 3, 4, 5, 6, 7, 8]
+        ]
+
+
+    def test_squad_with_unwordpieceable_passage(self):
+        # pylint: disable=line-too-long
+        tokenizer = WordTokenizer()
+
+        token_indexer = PretrainedBertIndexer("bert-base-uncased")
+
+        passage1 = "There were four major HDTV systems tested by SMPTE in the late 1970s, and in 1979 an SMPTE study group released A Study of High Definition Television Systems:"
+        question1 = "Who released A Study of High Definition Television Systems?"
+
+        passage2 = """Broca, being what today would be called a neurosurgeon, had taken an interest in the pathology of speech. He wanted to localize the difference between man and the other animals, which appeared to reside in speech. He discovered the speech center of the human brain, today called Broca's area after him. His interest was mainly in Biological anthropology, but a German philosopher specializing in psychology, Theodor Waitz, took up the theme of general and social anthropology in his six-volume work, entitled Die Anthropologie der Naturvölker, 1859–1864. The title was soon translated as "The Anthropology of Primitive Peoples". The last two volumes were published posthumously."""
```

Comment:
You're using triple quotes already; can you make this into multiple shorter lines instead of one super long one?  Same with the passage above.